src/app/components/nav/dash-footer/dash-footer.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, Inject, OnInit} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+@Component({
+  selector: 'app-dash-footer',
+  templateUrl: './dash-footer.component.html',
+  styleUrls: ['./dash-footer.component.scss']
+})
+export class DashFooterComponent implements OnInit {
+
+  year: any;
+
+  constructor(@Inject(DOCUMENT) private document: Document) { }
+
+  ngOnInit() {
+    this.year = new Date().getFullYear();
+    const js = this.document.createElement('script');
+    js.src = 'https://buttons.github.io/buttons.js';
+    this.document.getElementsByTagName('head')[0].appendChild(js);
```

Comment:
We should avoid using plain javascript in our Angular application. Use typescript and Angular features.

Reply:
@sanketbansal  so you want to add buttons.js to use class `github-button` , right ?

Reply:
You can simply add script tag in html file. But I don't think you should  import whole js file just to get github icon. Isn't we are already using font-awesome that has github stars and fork icons?

Reply:
it's not only github icons actually it is used to fetch number of stars and forks on the github repository also.

Reply:
> Just want to let you know: If you want to manipulate any DOM element (the similar thing that you were doing -- accessing any tag) Angular have `ElementRef `, `ViewRef`, .. reference types to pick any DOM element and modify it. There is `Renderer` to create new elements as well.

Thanks for letting me know. I tried that but I can not get access to head tag do not know why?
Maybe because it's not a component specific element. what do you think?